docs/concepts/workloads/controllers/deployment.md

```diff
@@ -288,6 +288,8 @@ It is generally discouraged to make label selector updates and it is suggested t
 In any case, if you need to perform a label selector update, exercise great caution and make sure you have grasped
 all of the implications.
 
+In Kubernetes 1.8 or later, after a Deployment gets created, its label selector is immutable.
```

Comment:
This is only true for `apps/v1beta2` Deployments. You can still mutate selectors of `extensions/v1beta1` and 1apps/v1beta1` Deployments. Same as other behavior changes described below.

Reply:
Done.